src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -577,6 +608,12 @@ public GhprbGitHubAuth getGitHubAuth(String gitHubAuthId) {
         // map of jobs (by their fullName) and their map of pull requests
         private Map<String, ConcurrentMap<Integer, GhprbPullRequest>> jobs;
         
+        /**
+         *  map of jobs (by the repo name);  No need to keep the projects from shutdown to startup.
+         *  New triggers will register here, and ones that are stopping will remove themselves.
+         */
+        private transient Map<String, Set<AbstractProject<?, ?>>> repoJobs;
```

Comment:
That is correct since it is part of the descriptor which is a static member of the trigger class.